submissions/dummies/estimator.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pandas as pd
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.compose import make_column_transformer
+from sklearn.preprocessing import OrdinalEncoder
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+
+def _merge_external_data(X):
+    filepath = os.path.join(os.path.dirname(__file__),
+                            'external_data_mini.csv')
```

Comment:
Don’t rename the file

Also external data are here part of the submission. You should not add any file to data folder

Reply:
whoops, that was because I copied the file from ramp workflow tests!

Reply:
Also I added a file to the data folder so that I can demonstrate the merge within the notebook but you're right that I can just use the external data file within the starting kit submission in the notebook